consumer/apis/dropbox/oauth_20_dropbox_api.e

```diff
@@ -0,0 +1,66 @@
+note
+	description: "Summary description for {OAUTH_20_DROPBOX_API}."
+	date: "$Date$"
+	revision: "$Revision$"
+	EIS: "name=Dropbox OAuth 2.0", "src=https://www.dropbox.com/developers/blog/45/using-oauth-20-with-the-core-api", "protocol=url"
+class
+	OAUTH_20_DROPBOX_API
+
+inherit
+
+	OAUTH_20_API
+		redefine
+			access_token_extractor,
+			access_token_verb
+		end
+
+feature -- Access
+
+	access_token_extractor: ACCESS_TOKEN_EXTRACTOR
+		do
+			create {JSON_TOKEN_EXTRACTOR} Result
+		end
+
+	access_token_verb: READABLE_STRING_GENERAL
+		do
```

Comment:
missing description, and I guess we should never use READABLE_STRING_GENERAL for the result.
Either use STRING_8 or STRING_32 
eventually IMMUTABLE_STRING_32